test/static_dsl.jl

```diff
@@ -97,7 +103,7 @@ params = ir.arg_nodes[2]
 
 # is_outlier
 is_outlier = ir.choice_nodes[1]
-@test is_outlier.name == :is_outlier
+# @test is_outlier.name == :is_outlier
```

Comment:
@ztangent The name is now a gensym right? If so, it's fine to delete these checks. We should probably clarify in the IR code what the purpose of the name actually is (for variable binding).

Reply:
Yup, it's a gensym now -- will delete these checks!